pkgs/development/mobile/androidenv/examples/shell.nix

```diff
@@ -0,0 +1,107 @@
+{
+  pkgs ? import ../../../../.. {},
+  pkgs_i686 ? import ../../../../.. { system = "i686-linux"; },
```

Comment:
```suggestion
  # if you copy the example out of nixpkgs, 
  # you want to use this line instead of the next line:
  #pkgs ? import <nixpkgs> {},
  pkgs ? import ../../../../.. {},
  pkgs_i686 ? import pkgs.pkgsi686Linux { system = "i686-linux"; },
```